src/CommandingOfficers/Modifiers/COModifier.java

```diff
@@ -1,70 +0,0 @@
-package CommandingOfficers.Modifiers;
-
-import java.io.Serializable;
-import java.util.ArrayList;
-
-import CommandingOfficers.Commander;
-import Units.UnitModel;
-
-public interface COModifier extends Serializable
```

Comment:
At least two implementing classes have now been orphaned now that this is gone. I probably would have kept this around (along with the handling code in `Commander`) so long as valid subclasses exist. Thoughts?

Reply:
I would much rather `CommanderAbility`s handle their side-effects on permanent state explicitly - the way it was before created too much room to call the wrong thing and end up with unwanted permanent changes (see: Ave's stacking attack boost bug).
The two remaining cases (production modifier and temporary transformation) are honestly super dissimilar to the point where I'm not sure having a common interface for them makes a lot of sense.

Reply:
OK, fair enough.